docs/core/StrategyManager.md

```diff
@@ -158,12 +192,26 @@ function forceTotalWithdrawal(address staker) external
     onlyWhenNotPaused(PAUSED_WITHDRAWALS)
     onlyNotFrozen(staker)
     nonReentrant
-    returns (bytes32)
+    returns (IStrategy[] memory, uint256[] memory, bytes32)
 ```
 
-**Effects**:
+The `DelegationManager` calls this method when a Staker is undelegated from an Operator. If the Staker has shares in any strategies, this method removes the shares from the Staker's balance and creates a `QueuedWithdrawal` for the shares and strategies in question. The Staker must wait for `withdrawalDelayBlocks` before they can complete the withdrawal.
```

Comment:
one thing to note, although it's implicit here: the 'withdrawer' for this forced withdrawal is the staker themself